compiler/ras/Debug.cpp

```diff
@@ -3615,8 +3615,9 @@ TR_Debug::dump(TR::FILE *pOutFile, TR_CHTable * chTable)
          char guardKindName[38];
          sprintf(guardKindName, "%s %s", getVirtualGuardKindName((*info)->getKind()), (*info)->mergedWithHCRGuard()?"+ HCRGuard ":"");
 
-         if ((*info)->getKind() == TR_OSRGuard)
-            trfprintf(pOutFile, "[%4d] %-38s OSRGuard\n", i, guardKindName);
+         if (!(*info)->getSymbolReference())
+            trfprintf(pOutFile, "[%4d] %-38s %s%s\n",
+                  i, guardKindName ,(*info)->isInlineGuard()?"inlined ":"", guardKindName);
```

Comment:
The prior behavior, which applied to only OSR Guards, was to repeat the guardKindName in the place of the callee symbol. This function is printing out a table, so I believe it is desirable to not leave a column empty.